src/concolic/concolic_wasm_ffi.ml

```diff
@@ -69,6 +69,19 @@ module M :
         let n = Float64.of_bits n in
         (F64 n, Value.pair n (Expr.mk_symbol sym)) )
 
+  let symbol_bool () : Value.int32 Choice.t =
+    Choice.with_new_symbol (Ty_bitv 32) (fun sym forced_value ->
+        let n =
+          match forced_value with
+          | None -> Int32.logand 1l (Random.bits32 ())
+          | Some (Num (I32 n)) -> n
+          | _ -> assert false
+        in
+        let sym_expr =
+          Expr.make (Cvtop (Ty_bitv 32, Zero_extend 31, Expr.mk_symbol sym))
+        in
+        (I32 n, Value.pair n sym_expr) )
```

Comment:
@filipeom what do you think about this ?

Reply:
This won't work because of the: 

```ocaml
Choice.with_new_symbol (Ty_bitv 32) (fun sym forced_value ->
```

At the beginning of the function, so It will create a 63-bit vector. For this to be correct we would need to create a `Ty_bitv 1` symbol. But I don't think it's supported atm :sweat_smile: 

We can probably get away with just `let sym expr = Expr.mk_symbol sym in`